ompi/mca/coll/han/coll_han_reduce.c

```diff
@@ -172,7 +172,8 @@ mca_coll_han_reduce_intra(const void *sbuf,
         mca_coll_task_t *t_next_seg = OBJ_NEW(mca_coll_task_t);
         /* Setup up t_next_seg task arguments */
         t->cur_task = t_next_seg;
-        t->sbuf = (char *) t->sbuf + extent * t->seg_count;
+        if (t->sbuf != MPI_IN_PLACE)
```

Comment:
We use curly braces around if bodies (even single line bodies), like all rational thinking people.

Reply:
Thank you! Fixed in new revision.